wordpress-mobile/WordPress-Android
# Jetpack focus full screen bottom sheet

This PR updates bottom sheet to full screen to be shown on **Stats**, **Reader** and **Notifications** screens.  It is also updated to handle dark theme and landscape modes.

NOTE:  At the moment it appears once on Reader and Notfications and every time on Stats screens for testing purpose only.  This logic will be update in separate PR after confirmation and also hook up to all the badges/banners where required in the next sprint.

| Full screen | Dark theme | Landscape |
|---|---|---|
|![Screenshot_20220718_172201](https://user-images.githubusercontent.com/990349/179466429-5a48205b-982b-4023-b48f-ab3d58897417.png)|![Screenshot_20220718_173437](https://user-images.githubusercontent.com/990349/179466478-c040a520-985f-4233-94a8-62a24d560a19.png)|![Screenshot_20220718_174412](https://user-images.githubusercontent.com/990349/179466717-ddb96521-93f2-455c-bd33-588b65319c9d.png)|


To test:

Setup:

- Launch WordPress app
- Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings)
- Select Debug Settings
- Find **JetpackPoweredFeatureConfig** under Features in development as shown on this PR #16908
- Re-launch app (scroll down if necessary)

Test 1:

- Tap on **Reader** tab at the bottom, Ensure a bottom sheet appears full screen as shown in the image above 
- Ensure secondary button reads 'Continue to Reader'
- Observe Logo animation 

Test 2:

- Return to My Site
- Tap on **Notifications** tab at the bottom, Ensure a bottom sheet appears full screen 
- Ensure secondary button reads 'Continue to Notifications'

Test 3:

- Return to My Site
- Tap on **Stats** button/menu, Ensure a bottom sheet appears full screen 
- Ensure secondary button reads 'Continue to Stats'

Test 4:

- Turn on Dark theme
- Follow steps in Test 3 
- Ensure it shows as in the second image above

Test 5:

- Return to My Site
- Tap on **Stats** button/menu, Ensure a bottom sheet appears full screen 
- Turn device to landscape
- Ensure the bottom sheet appears as show above in the third image

Test 6:
- Return to My Site
- Make sure the Jetpack powered badge appears on the home tab as shown on this PR #16901 
- Tap on the **Jetpack powered** badge, Ensure a bottom sheet appears as shown in video on this PR #16910 


## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested different

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
